jmxtrans-output/jmxtrans-output-core/src/main/java/com/googlecode/jmxtrans/model/output/TCollectorUDPWriter.java

```diff
@@ -68,7 +68,7 @@ public TCollectorUDPWriter(
 			@JsonProperty("tagName") String tagName,
 			@JsonProperty("mergeTypeNamesTags") Boolean mergeTypeNamesTags,
 			@JsonProperty("metricNamingExpression") String metricNamingExpression,
-			@JsonProperty("addHostnameTag") Boolean addHostnameTag,
+			@JsonProperty("addHostnameTag") boolean addHostnameTag,
```

Comment:
I know, leaving it to coercion doesn't feel as clean.  Good news being is that the class is on the deprecation list 👍 , now to work on gaining adoption of the Factory.